get-protoc.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# 
+# Gets the version of protoc needed to build Closure, and prints its path.
+# Downloads sources and builds them if needed, and caches the binary in ~/.protobuf.
+#
+# Note: the version of protobuf-compiler available through
+# Travis CI's addons.apt.packages mechanism is 2.4.1, and protoc's version really
+# needs to match that of the protobuf-java dependency.
+#
+set -eu
+
+PROTOBUF_VERSION=$( \
+  cat pom-main.xml | \
+  grep "<protobuf.version>" | \
+  sed -E 's/.*>(.*)<.*/\1/' \
+)
+
+# Where we'll install protoc (on Travis CI, will be cached).
+PROTOBUF_DIR=$HOME/.protobuf/$PROTOBUF_VERSION
+PROTOC=$PROTOBUF_DIR/bin/protoc
+
+function getAndBuildProtoc() {
+  local name=protobuf-$PROTOBUF_VERSION
+
+  rm -fR $name*
+  wget https://protobuf.googlecode.com/files/$name.tar.gz
```

Comment:
(assuming we do need it) is this the right URL to get the latest version? I would imagine there is a dl.google.com URL that is more up to date.